Sources/OpenAI/Public/Models/ChatStreamResult.swift

```diff
@@ -149,4 +148,18 @@ public struct ChatStreamResult: Codable, Equatable, Sendable {
         case systemFingerprint = "system_fingerprint"
         case usage
     }
+    
+    public init(from decoder: any Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let parsingOptions = decoder.userInfo[.parsingOptions] as? ParsingOptions ?? []
+        
+        self.id = try container.decodeString(forKey: .id, parsingOptions: parsingOptions)
```

Comment:
```suggestion
self.id = try container.decodeIfPresent(String.self, forKey: .id)
```

This needs to be optional or default to a `UUID()`